Newer GCCs reduced dependency between header files, so it is
neccessary to add the cstring header here manually

--- lineak-defaultplugin-0.9.orig/default_plugin/defaultplugin.cpp
+++ lineak-defaultplugin-0.9/default_plugin/defaultplugin.cpp
@@ -35,6 +35,7 @@
 #include <iostream>
 #include <sstream>
 #include <stdlib.h>
+#include <cstring>
 using namespace std;
 using namespace lineak_core_functions;
 
Newer GCCs reduced dependency between header files, so it is
neccessary to add the cstring header here manually

--- lineak-defaultplugin-0.9.orig/default_plugin/defaultplugin.cpp
+++ lineak-defaultplugin-0.9/default_plugin/defaultplugin.cpp
@@ -35,6 +35,7 @@
 #include <iostream>
 #include <sstream>
 #include <stdlib.h>
+#include <cstring>
 using namespace std;
 using namespace lineak_core_functions;
 
